backend/openapi.yaml

```diff
@@ -4881,22 +5097,46 @@ components:
           example: 42
         credentials_label:
           type: string
-          example: "ogp-twilio-creds"
+          example: 'ogp-twilio-creds'
         recipient:
           type: string
-          example: "98765432"
+          example: '98765432'
         body:
           type: string
-          example: "Hello world"
-        message_id:
```

Comment:
just to clarify, the idea here is we'll still store this in our db, but we won't be returning this to the user? why or why not? (I was thinking with the `message_id` in theory they can go to Twilio to get more info from them?)

Reply:
I can guarantee u no one will go directly to Twilio

Reply:
fair enough